addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2453,6 +2453,15 @@ options.registry.ContainerWidth = options.Class.extend({
             this.$target.addClass('o_container_preview');
         }
     },
+     /**
+     * @override
+     */
+    _computeWidgetVisibility: function () {
+        if (this.$target.closest('#o_wblog_post_content').length) {
+            return false
+        }
+        return this._super(...arguments);
+    },
```

Comment:
In this case you might be able to simply avoid instanciating this option by adding the right selector to the data-exclude attribute instead of simply hiding it with `_computeWidgetVisibility`